skype/skype-ps/skype/New-CsHuntGroup.md

```diff
@@ -261,16 +264,16 @@ Applicable: Skype for Business Online
 
 Required: False
 Position: Named
-Default value: None
+Default value: Disconnect
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -OverflowActionTarget
-PARAMVALUE: Uri
+### -TimeoutActionTarget
+The TimeoutActionTarget represents the target of the timeout action. For example, if the TimeoutAction is set to Forward, this parameter might be set to a SIP address or a PSTN phone number. This field is optional unless the TimeoutAction is set to Forward, which requires a SIP address.
```

Comment:
We should not mention target can be PSTN number

Reply:
Rephrase: "The TimeoutActionTarget represents the target of the timeout action. If the TimeoutAction is set to Forward, this parameter must be set to a SIP address. Otherwise, this parameter is optional."

Reply:
Fixed